mimic/test/test_flavors.py

```diff
@@ -139,3 +137,12 @@ def test_get_flavor_list_after_list_flavors(self):
             get_server_flavor_response_body = self.get_server_flavor('/flavors/' + flavor_id)
             self.assertEqual(flavor_id, get_server_flavor_response_body['flavor']['id'])
             self.assertEqual(name, get_server_flavor_response_body['flavor']['name'])
+
+    def test_get_OnMetal_flavor(self):
```

Comment:
I'd prefer if this was a pure refactor that's able to maintain coverage without adding any new tests. I'll let it slide this time, though.